### ISSUE TITLE
Admin UI: Removing contents from multi-select field has no effect

### ISSUE THREAD

**Issue by intmainvoid** [exogee-technology/graphweaver#243]
**Describe the bug**
When removing data from a field that is a relationship field to another field, Graphweaver Admin UI reports that the change has been saved when in fact the change has not been persisted.

**To Reproduce**

1. Create a new Postgres database populating the DB with the schema with the supplied DDL (see below)
2. Add some sample records to the 3 tables
3. Create a new project using the command: `npx graphweaver@latest init`
4. Follow the prompts to create a new GW project
5. Change current directory to the project directory
6. Run the command: `npx graphweaver@latest import postgresql` to kick off the Postgres importer
7. Follow the prompts to import the specified database
8. Wait until it completes
9. Start Graphweaver by running `pnpm start`
10. Navigate to `http://localhost:9000/` using the browser
11. Click on the `Voyage` entity to display the Voyage records (which you have hopefully added in the step above)
12. Click on one of the voyage records that have the `exporter_voyages` column populated. This will open the side panel
13. Observe that - in the field `exporter-voyages` - there is "1 Selected" (or however many there are selected).
14. Remove the selected items in the field.
15. Save the record
16. Refresh the browser window
17. Observe that the requested change has not been saved.

**Expected behavior**
When removing the related record in Postgres, that field is then set to `null`.

**Actual behavior**
Removing the field's contents and saving the record has no effect even though the GW Admin UI states that the operation was successful

**Screenshots or Screencasts**
https://github.com/exogee-technology/graphweaver/assets/719151/f4dae040-fff3-413a-adfb-e45a8d22f66d

**Desktop (please complete the following information):**

- OS and version: MacOS 13.5.2 (22G91)
- Postgres version: Version 2.6.8a (228)
- Node: v20.10.0
- pnpm: 8.10.5

**Additional context**
DDL to create the database:
```
CREATE TABLE voyage (
    id BIGSERIAL PRIMARY KEY,
    name character varying(255) NOT NULL
);

CREATE TABLE exporter (
    id BIGSERIAL PRIMARY KEY,
    company character varying(255) NOT NULL
);

CREATE TABLE exporter_voyages (
    id BIGSERIAL PRIMARY KEY,
    exporter_id bigint,
    voyage_id bigint REFERENCES voyage(id) ON DELETE CASCADE ON UPDATE CASCADE
);
```

**Comment by agaw** [comment]
This was fixed in https://github.com/exogee-technology/graphweaver/pull/255, once this makes it to the next public release I'll retest and close this issue.

**Comment by agaw** [comment]
Confirmed this is fixed in 0.2.13.
